modules/appeals_api/spec/workers/appeal_received_job_spec.rb

```diff
@@ -5,6 +5,85 @@
 # rubocop:disable Metrics/ModuleLength
 module AppealsApi
   RSpec.describe AppealReceivedJob, type: :job do
+    describe 'va notify vet email templates' do
+      before do
+        @dc = described_class.new
```

Comment:
Generally speaking, rspec strongly encourages use of `let` notation over instance variables. Something like this should work:

```ruby
describe 'va notify vet email template' do
  let(:job) { described_class.new }
  let(:client) { instance_double(VaNotify::Service) }
  let(:opts) { { 'email_identifier' => { 'id_value' => 'fake_email@email.com', 'id_type' => 'email' } }
  
  before do
    allow(VaNotify::Service).to receive(:new).and_return(client)
    allow(client).to receive(:send_email)
  end
```

Additionally, setting instance variables within tests feels like the test knows too much about the details of the class. I prefer to interact with the classes through their public interfaces, and in this case the `perform` method is the most relevant. I'm actually unsure why the `*_received` methods are public - an oversight or a holdover from removing the eventing system? - but I think testing the HLR/NOD/SC received can be done via `job.perform(opts.merge('receipt_event': 'hlr_received'))`, etc.

All that said, I'm not sure we should hold off on merging the fix simply because of spec style choices, so let's just circle around in another branch so that we can get emails sending properly sooner than later. Good find!

Reply:
As suggested, fixed the test per drews comments above in a later pr